l10n.toml

```diff
@@ -0,0 +1,87 @@
+
+basepath = "."
+
+locales = [
+    "am",
+    "ar",
+    "ast",
+    "az",
+    "bg",
+    "bn-BD",
+    "bn-IN",
+    "bs",
+    "ca",
+    "cak",
+    "cs",
+    "cy",
+    "da",
+    "de",
+    "dsb",
+    "el",
+    "eo",
+    "es-AR",
+    "es-CL",
+    "es-ES",
+    "es-MX",
+    "eu",
+    "fa",
+    "fi",
+    "fr",
+    "fy-NL",
+    "ga-IE",
+    "hi-IN",
+    "hsb",
+    "hu",
+    "hy-AM",
+    "ia",
+    "in",
+    "it",
+    "iw",
+    "ja",
+    "ka",
+    "kab",
+    "kk",
+    "ko",
+    "lo",
+    "lt",
+    "meh",
+    "mix",
+    "ms",
+    "my",
+    "nb-NO",
+    "ne-NP",
+    "nl",
+    "nn-NO",
+    "pl",
+    "pt-BR",
+    "ro",
+    "ru",
+    "sk",
+    "sl",
+    "sq",
+    "sr",
+    "sv-SE",
+    "ta",
+    "te",
+    "th",
+    "tr",
+    "trs",
+    "uk",
+    "ur",
+    "vi",
+    "zam",
+    "zh-CN",
+    "zh-TW",
+]
+
+# Expose the following branches to localization
+# Changes to this list should be announced to the l10n team ahead of time.
+branches = [
+    "master",
+]
+
+[env]
+
+[[paths]]
+  reference = "app/src/main/res/values/strings.xml"
```

Comment:
It's unlikely, but if we support multiple gradle modules in the same repository that ship their own `strings.xml` (e.g. `app/src/.../strings.xml` & `support/src/.../strings.xml`), is that supported?

Reply:
Yes. We do that in a-c, https://github.com/mozilla-mobile/android-components/blob/c3cae262766d1f288bab81ac6f9db9ffd5d33dbd/l10n.toml#L23. If you end up needing, that, I'm happy to help or review.